Expecto.netcore/Expecto.netcore.fsproj

```diff
@@ -1,54 +1,33 @@
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
+
   <PropertyGroup>
     <AssemblyName>Expecto</AssemblyName>
     <Version>1.1.2</Version>
```

Comment:
The `Version` attribute is not stricly needed in `fsproj`. Was needed before in preview4 (because nuget and prerelease versions packages)
You can pass it as msbuild property, like `dotnet pack /p:Version=1.1.2` so is easier to configure from build script (no need to maintan in fsprj)